microdonuts/src/main/java/com/otsample/api/App.java

```diff
@@ -100,17 +99,17 @@ static boolean configureGlobalTracer(Properties config, String componentName)
                 .spanReporter(reporter)
                 .build());
         } else if ("lightstep".equals(tracerName)) {
-            Options opts = new Options.OptionsBuilder()
-                .withAccessToken(config.getProperty("lightstep.access_token"))
-                .withCollectorHost(config.getProperty("lightstep.collector_host"))
-                .withCollectorPort(Integer.decode(config.getProperty("lightstep.collector_port")))
-                .withComponentName(componentName)
-                .build();
-            tracer = new JRETracer(opts);
+            OpenTelemetryConfiguration.newBuilder()
+                .setServiceName(componentName)
+                .setAccessToken(config.getProperty("lightstep.access_token"))
```

Comment:
It would be nice to directly use the new system property names, e.g. `ls.acess.token` (as also, in the future, we want to support configuration through property files).

Reply:
updated to use system properties